content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -32,54 +31,80 @@ the product page, you can see the following sections:
 When you refresh the page, the app shows different versions of reviews in the product page.
 The app presents the reviews in a round robin style: red stars, black stars, or no stars.
 
-## Enabling Istio authorization
+## Enforcing mesh-level access control
 
-Run the following command to enable Istio authorization for the `default` namespace:
+Using Istio authorization, you can easily setup mesh-level access control for all workloads in the mesh.
 
-{{< text_import file="examples/TestAuthorizationForHTTPServices.txt" snippet="enabling_istio_authorization.sh" >}}
+Run the following command to create a mesh-level access control policy:
 
-Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). Now you should see
-`"RBAC: access denied"`. This is because Istio authorization is "deny by default", which means that you need to
-explicitly define access control policy to grant access to any service.
+{{< text bash >}}
```

Comment:
These are embedded right in the task page. We have tests for the old task page but we don't have one for this for now.

It's more convenient for users to try out the new policy, they don't need access to local files and just need to copy and run the command. This technique is also used in many other tasks, like authentication and networking.

The other benefit is it reduces the maintenance burden, we could only maintain one copy of the sample policy.

Reply:
I know we're trying to automate the test and I wrote the automatic test for the HTTP page in istio.

The command (kubectl apply) is used everywhere in istio.io and for any normal user, I don't think it's causing any problems for them to understand the command which is separate from the policy being applied.

> It is also a lot easier for them to copy paste a command that references a sample file and it is less error prone. Not all readers use the copy button.

You need to use the copy button even if it's referring to a sample file. Also it's very common for user to actually look at the contents of the yaml, if it's a link, the user have to do one more click to open the page and switch back to see the corresponding explanation of the yaml.

> Lastly and regarding maintaining the samples: it is better and easier to maintain the samples in the repository than trying to maintain the samples embedded in the content.

From my experience of maintaining the task and sample, it is harder to maintain them in separate repos because if you want to make some change to the sample, you usually need to do it separately in two PRs, one to update the sample and the other to update the doc, there is a gap here and could also introduce inconsistent between the doc and the sample because they are updated separately.

Also this is not about code freeze, the "code freeze" is actually one of the examples of how it is making the maintenance harder, we have to keep the two repo in sync which is just hard to maintain.